src/classes/treeview/WebResourcesProvider.ts

```diff
@@ -15,18 +17,33 @@ export class WebResourcesProvider implements vscode.TreeDataProvider<WebResource
 
     async getChildren(element?: WebResource): Promise<WebResource[]> {
         if (element) {
-            //I think we're at the end of a branch of the tree? Just return nothing
-            return Promise.resolve([]);
+            if (element.pathOnDisk) {
+                return Promise.resolve([new WebResource(element.pathOnDisk, element.id + "idk", true,
+                    vscode.TreeItemCollapsibleState.Collapsed
+                )]);
+
+            } else {
+                return Promise.resolve([]);
+            }
         } else {
-            //I think we're at the root of the tree? Just return the list
-
             var webresources = await this.syncer.retreiveWebResourcesInSolution(this.solutionName);
-            
-            var mapped = webresources.map(r => new WebResource(r.name, r.id, true,
-                vscode.TreeItemCollapsibleState.Collapsed
-            ));
 
-            return Promise.resolve(mapped);
+            var mapped = Promise.all(webresources.map(async r => {
+                var webResource = new WebResource(r.name, r.id, true,
+                    vscode.TreeItemCollapsibleState.Collapsed
+                );
+
+                var pathInPAWithoutPublisher = r.name.split("_")[1];
```

Comment:
Maybe revise this to something like x.substring(x.indexOf("\_")) in case a web resource has other underscores in it's path, like new_/form_scripts/main_account.js